File: examples/crypto/news/go/getCryptocurrencyNews.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/crypto"
)

// Get news related to a currency
func GetCryptocurrencyNews() {
	cryptoService := crypto.NewCryptoService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := cryptoService.News(&crypto.NewsRequest{
		Symbol: "BTCUSD",
	})
	fmt.Println(rsp, err)
}

File: examples/js/crypto/news/getCryptocurrencyNews.js
const { CryptoService } = require("m3o/crypto");

// Get news related to a currency
async function getCryptocurrencyNews() {
  let cryptoService = new CryptoService(process.env.MICRO_API_TOKEN);
  let rsp = await cryptoService.news({
    symbol: "BTCUSD",
  });
  console.log(rsp);
}

getCryptocurrencyNews();
